pom.xml

```diff
@@ -2068,12 +2068,8 @@
                 <artifactId>pac4j-oidc</artifactId>
                 <version>${pac4j.version}</version>
                 <exclusions>
-                    <exclusion>
-                        <groupId>com.sun.mail</groupId>
-                        <artifactId>javax.mail</artifactId>
-                    </exclusion>
-		    <exclusion>
-		        <groupId>org.mockito</groupId>
+		            <exclusion>
```

Comment:
nit: spacing is off

Reply:
Turns out git UI does not handle spaces and tabs well. My IDE inserted tabs which messed up git UI. I'll change it to spaces.